src/components/popover/popover.tsx

```diff
@@ -138,6 +138,7 @@ export class Popover {
         const propertyNames = [
             '--popover-surface-width',
             '--popover-body-background-color',
+            '--popover-border-radius',
```

Comment:
I'm guessing this doesn't actually fix Lundalogik/crm-feature#1902, and that you just added that in order to link the PR to the issue? (Which is totally ok, of course! I'm just double-checking 🙂)

Reply:
Ah! I just read the comment on the issue. Sorry! 😄